app/org/sagebionetworks/bridge/dynamodb/DynamoUpload.java

```diff
@@ -17,8 +17,9 @@
 import org.joda.time.DateTimeZone;
 import org.joda.time.LocalDate;
 
+@DynamoThroughput(writeCapacity=50, readCapacity=50)
 @DynamoDBTable(tableName = "Upload")
-public class DynamoUpload implements Upload, DynamoTable {
+public class DynamoUpload implements Upload {
```

Comment:
This is safe, but probably not necessary, since no one uses DynamoUpload anymore. (I'll need to delete it in a future code change.) Everything goes through DynamoUpload2 now.